frappe/database/database.py

```diff
@@ -310,7 +310,7 @@ def _log_query(
 
 		if frappe.conf.logging == 2:
 			_query = _query or str(mogrified_query)
-			frappe.log(f"<<<< query\n{_query}\n>>>>")
+			frappe.log(f"#### query\n{_query}\n####")
```

Comment:
this isn't in `develop`

Reply:
Yeah kinda unrelated. `<<<` shows up frequently while searching for git conflicts, adding to develop too.